core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1022,45 +1022,63 @@
       "Amount of time for a single replication RPC call to last before failing"
           + " the attempt. See replication.work.attempts."),
   // Compactor properties
+  @Experimental
   COMPACTOR_PREFIX("compactor.", null, PropertyType.PREFIX,
       "Properties in this category affect the behavior of the accumulo compactor server."),
+  @Experimental
   COMPACTOR_PORTSEARCH("compactor.port.search", "true", PropertyType.BOOLEAN,
```

Comment:
The default for port search should be false to be consistent with other services that have this feature.

Reply:
Changed default from "true" to "false" in 42d0c8445d964f079f39ef4c50d05559191982a3